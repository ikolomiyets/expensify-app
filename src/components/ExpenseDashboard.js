import React from 'react';

const ExpenseDashboard = () => (
    <div className="container">
        <div className="h1">React JS Starter Project</div>
        <p>Font Awesome Icon <i className="fa fa-snowflake-o"></i></p>
    </div>
);

export default ExpenseDashboard;