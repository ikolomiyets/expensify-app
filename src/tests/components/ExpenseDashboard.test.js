import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboard';

test('expect expense dashboard to render', () => {
   const wrapper = shallow(<ExpenseDashboard/>);
   expect(wrapper).toMatchSnapshot();
});