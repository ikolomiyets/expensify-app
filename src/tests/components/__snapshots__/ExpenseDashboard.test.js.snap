// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect expense dashboard to render 1`] = `
<div
  className="container"
>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
